src/core_plugins/kibana/public/management/sections/objects/_objects.html

```diff
@@ -1,80 +1,179 @@
-<kbn-management-app section="kibana">
-  <kbn-management-objects class="container-fluid">
-    <div class="header">
-      <h2 class="title">Edit Saved Objects</h2>
-      <button class="btn btn-default controls" ng-click="exportAll()"><i aria-hidden="true" class="fa fa-download"></i> Export Everything</button>
-      <file-upload on-read="importAll(fileContents)" upload-selector="button.upload">
-        <button class="btn btn-default controls upload" ng-click>
-          <i aria-hidden="true" class="fa fa-upload"></i> Import
+<kbn-management-app section="kibana" class="kuiView">
+  <kbn-management-objects class="kuiViewContent kuiViewContent--constrainedWidth">
+    <!-- Header -->
+    <div class="kuiViewContentItem kuiSubHeader">
+      <h1 class="kuiTitle">
+        Edit Saved Objects
+      </h1>
+
+      <div>
+        <button
+          class="kuiButton kuiButton--basic kuiButton--iconText"
+          ng-click="exportAll()"
+        >
+          <span aria-hidden="true" class="kuiButton__icon kuiIcon fa-download"></span>
+          Export Everything
         </button>
-      </file-upload>
+
+        <file-upload
+          on-read="importAll(fileContents)"
+          upload-selector="button.upload"
+        >
+          <button class="kuiButton kuiButton--basic kuiButton--iconText">
+            <span aria-hidden="true" class="kuiButton__icon kuiIcon fa-upload"></span>
+            Import
+          </button>
+        </file-upload>
+      </div>
     </div>
-    <p>
+
+    <!-- Intro -->
+    <p class="kuiViewContentItem kuiVerticalRhythm">
```

Comment:
What is the significance of `VerticalRhythm` in `kuiVerticalRhythm`? It doesn't seem to be descriptive in the same way as `SubHeader` might be in `kuiSubHeader `, for instance. @